src/components/features/ProductSearch/ProductSearch.module.scss

```diff
@@ -24,14 +24,69 @@
       right: 10px;
       z-index: 0;
     }
+  }
 
-    select {
-      -webkit-appearance: none;
-      border: 0;
-      background-color: transparent;
-      padding: 5px 30px 5px 35px;
-      position: relative;
+  .category:hover {
+    background-color: $primary;
+    color: #ffff;
+
+    .icon:first-child {
+      color: inherit;
+    }
+    .icon:last-child {
+      transform: rotate(180deg);
+    }
+
+    .list {
+      height: auto;
       z-index: 1;
+      border: 1px solid $form-border-color;
+      left: -1px;
+      right: -1px;
+      
+      a {
+        height: 45px;
+        font-size: 14px;
+        padding: 10px 20px;
+        color: $text-color;
+        transition: .5s ease-in-out;
+       
+        &:hover {
+          background-color: $primary;
+          color: #ffff;
+          padding-left: 80px;
```

Comment:
You store most of the colors as SCSS variables, so why have you left this one? ;)

Reply:
didn't wont to interfere in the other task that handles the color variables in the settings.scss. I will correct it.

Reply:
I add a new variable in the settings.scss and used it in ProductSearch component. have a nice Saturday :)